.circleci/config.yml

```diff
@@ -1336,6 +1336,8 @@ jobs:
           steps:
             - store_artifacts:
                 path: ./op-acceptance-tests/logs
+      - notify-failures-on-develop:
+          mentions: "@changwan @Anton Evangelatov"
```

Comment:
Not sure this will work since there is whitespace inside the name. We may use a user id, or create a slack group like `"@proofs-team"`